### PR TITLE
Docker compose (3 of 3)

### PR DESCRIPTION
**Pull request by jmarcet**
Maintainer: Javier Marcet javier@marcet.info
Compile tested: x86-64 master
Run tested: x86-64 master

Description: third and final set of packages which complete docker-compose.
This PR depends on #11363 and #11364, and is the final part from splitting #11155

### CODE REVIEW

File: lang/python/python-jsonschema/Makefile [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-jsonschema
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=An implementation of JSON Schema validation for Python
```
Comment (BKPepe) [inline lang/python/python-jsonschema/Makefile]:
```suggestion
  TITLE:=An implementation of JSON Schema validation
```

File: lang/python/python-jsonschema/Makefile [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+PKG_NAME:=python-jsonschema
+PKG_VERSION:=3.2.0
+PKG_RELEASE:=1
+
+PYPI_NAME:=jsonschema
+PKG_HASH:=c8a85b28d377cc7737e46e2d9f2b4f44ee3c0e1deac6bf46ddefc7187d30797a
+
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=COPYING
+PKG_MAINTAINER:=Javier Marcet <javier@marcet.info>
```
Comment (BKPepe) [inline lang/python/python-jsonschema/Makefile]: Above PKG_LICENSE, please.

File: lang/python/python-jsonschema/Makefile [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=An implementation of JSON Schema validation for Python
+  URL:=https://github.com/Julian/jsonschema
+  DEPENDS:=+python3-light +python3-attrs +python3-six +python3-pyrsistent +python3-setuptools
+  VARIANT:=python3
+endef
+
+define Package/python3-jsonschema/description
+  An implementation of JSON Schema validation for Python
```
Comment (BKPepe) [inline lang/python/python-jsonschema/Makefile]:
```suggestion
  jsonschema is an implementation of JSON Schema for Python.
```

File: lang/python/python-jsonschema/Makefile [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-jsonschema
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=An implementation of JSON Schema validation for Python
+  URL:=https://github.com/Julian/jsonschema
+  DEPENDS:=+python3-light +python3-attrs +python3-six +python3-pyrsistent +python3-setuptools
```
Comment (BKPepe) [inline lang/python/python-jsonschema/Makefile]: Are you sure that python3-setuptools is required?

**Comment by jmarcet** [comment]
I added python3-distutils as a dependency since docker-compose breaks at runtime without it.

**Comment by peterwillcn** [comment]
ping @neheb

File: lang/python/docker-compose/Makefile [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+      +python3-docker \
+      +python3-dockerpty \
+      +python3-jsonschema \
+      +python3-pkg-resources \
+      +python3-requests \
+      +python3-setuptools \
+      +python3-six \
+      +python3-texttable \
+      +python3-websocket-client \
+      +python3-yaml
```
[the remark this answers is not in the document]
Comment (BKPepe) [reply lang/python/docker-compose/Makefile:43]: We shouldn't hurry. There were some changes 11 hours ago and they needs to be tested.

[the remark this answers is not in the document]
Comment (peterwillcn) [reply lang/python/docker-compose/Makefile:43]: @BKPepe Test pass.
Screen Shot 2020-04-20 at 17 45 42

[the remark this answers is not in the document]
Comment (BKPepe) [reply lang/python/docker-compose/Makefile:43]: That's just one package. What about others separately?

[the remark this answers is not in the document]
Comment (jmarcet) [reply lang/python/docker-compose/Makefile:43]: There is a small issue in pynacl which I just found out how to fix. Not sure it is the most optimal way. Please check #11364

**Comment by jmarcet** [comment]
docker-compose had an update to v1.26.0

This version introduced two new dependencies which where missing from OpenWRT: python3-distro & python3-dotenv.

The new packages are fairly simple although I'm not sure on python3-dotenv LICENSE, it is BSD but I am not sure which clause is the right one.

Everything is built and run tested on x86_64.

**Comment by jmarcet** [comment]
@jefferyto I've opened a [new PR](https://github.com/openwrt/packages/pull/12483) with all the changes.

**Pull request merged**